src/main/java/com/puppycrawl/tools/checkstyle/utils/CommonUtil.java

```diff
@@ -64,6 +64,9 @@
     public static final byte[] EMPTY_BYTE_ARRAY = new byte[0];
     /** Copied from org.apache.commons.lang3.ArrayUtils. */
     public static final double[] EMPTY_DOUBLE_ARRAY = new double[0];
+    /** Pseudo URL prefix pattern for loading from the class path. */
```

Comment:
`class path` -> classpath

Reply:
https://docs.oracle.com/javase/8/docs/technotes/tools/windows/classpath.html
Looks like both way is ok.